src/test/java/org/takes/rs/RsXsltXxeTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2014-2024 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.takes.rs;
+
+import java.io.IOException;
+import javax.xml.transform.stream.StreamSource;
+import org.cactoos.Text;
+import org.cactoos.io.InputStreamOf;
+import org.cactoos.text.Joined;
+import org.hamcrest.MatcherAssert;
+import org.junit.jupiter.api.Test;
+import org.llorllale.cactoos.matchers.Throws;
+
+/**
+ * Checking XXE vulnerability for XSLT transformer in response class {@link RsXslt}.
+ *
+ * @since 1.24.5
+ */
+final class RsXsltXxeTest {
```

Comment:
@PeJetuz why making a new class for this test? Why not placing it into the `RsXsltTest`?

Reply:
@yegor256 RsXsltTest class has 225 lines, adding 30 more lines is not very good in my opinion. Move the new test to RsXsltTest, will it be better?

Reply:
@PeJetuz yes, it's better to keep all test methods that belong to one class - in one `*Test` class, see https://www.yegor256.com/2023/01/19/layout-of-tests.html

Reply:
Thanks a lot, I'll fix it!

Reply:
@yegor256 I have fixed it, please review